test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -69,45 +86,66 @@ function waitForNodes {
     done
 }
 
-echo "Copying $IMG_NAME image to every node..."
-# Copy image to master
-scp -F ssh-config $SAVED_IMG k8s-node-master:/tmp/antrea-ubuntu.tar &
-pids[0]=$!
-# Loop over all worker nodes and copy image to each one
-for ((i=1; i<=$NUM_WORKERS; i++)); do
-    name="k8s-node-worker-$i"
-    scp -F ssh-config $SAVED_IMG $name:/tmp/antrea-ubuntu.tar &
-    pids[$i]=$!
-done
-# Wait for all child processes to complete
-waitForNodes "${pids[@]}"
-echo "Done!"
+function processAndPushImg {
+    IMG_NAME=$1
+    SAVED_IMG=$2
+
+    docker inspect $IMG_NAME > /dev/null
+    if [ $? -ne 0 ]; then
+        echo "Docker image $IMG_NAME was not found"
+        exit 1
+    fi
+
+    echo "Saving $IMG_NAME image to $SAVED_IMG"
+    docker save -o $SAVED_IMG $IMG_NAME
+
+    echo "Copying $IMG_NAME image to every node..."
+    # Copy image to master
+    scp -F ssh-config $SAVED_IMG k8s-node-master:/tmp/antrea-ubuntu.tar &
+    pids[0]=$!
+    # Loop over all worker nodes and copy image to each one
+    for ((i=1; i<=$NUM_WORKERS; i++)); do
+        name="k8s-node-worker-$i"
+        scp -F ssh-config $SAVED_IMG $name:/tmp/antrea-ubuntu.tar &
+        pids[$i]=$!
+    done
+    # Wait for all child processes to complete
+    waitForNodes "${pids[@]}"
+    echo "Done!"
+
+    echo "Loading $IMG_NAME image in every node..."
+    ssh -F ssh-config k8s-node-master docker load -i $SAVED_IMG &
+    pids[0]=$!
+    # Loop over all worker nodes and copy image to each one
+    for ((i=1; i<=$NUM_WORKERS; i++)); do
+        name="k8s-node-worker-$i"
+        ssh -F ssh-config $name docker load -i $SAVED_IMG &
+        pids[$i]=$!
+    done
+    # Wait for all child processes to complete
+    waitForNodes "${pids[@]}"
+    echo "Done!"
+}
 
-echo "Loading $IMG_NAME image in every node..."
-ssh -F ssh-config k8s-node-master docker load -i /tmp/antrea-ubuntu.tar &
-pids[0]=$!
-# Loop over all worker nodes and copy image to each one
-for ((i=1; i<=$NUM_WORKERS; i++)); do
-    name="k8s-node-worker-$i"
-    ssh -F ssh-config $name docker load -i /tmp/antrea-ubuntu.tar &
-    pids[$i]=$!
-done
-# Wait for all child processes to complete
-waitForNodes "${pids[@]}"
-echo "Done!"
+function copyManifest() {
+    YAML=$1
+    echo "Copying $YAML to every node..."
+    scp -F ssh-config $YAML k8s-node-master:~/ &
+    pids[0]=$!
+    # Loop over all worker nodes and copy image to each one
```

Comment:
s/image/manifest

Reply:
Typo was there in the earlier code. Fixed it.